executor/show_test.go

```diff
@@ -1759,3 +1759,38 @@ func TestShowBindingCache(t *testing.T) {
 	res = tk.MustQuery("show global bindings")
 	require.Equal(t, 2, len(res.Rows()))
 }
+
+func TestShowBindingCacheStatus(t *testing.T) {
+	store, clean := testkit.CreateMockStore(t)
+	defer clean()
+
+	tk := testkit.NewTestKit(t, store)
+	tk.MustExec("use test")
+	tk.MustQuery("show binding_cache status").Check(testkit.Rows(
+		"0 0 0 67108864"))
```

Comment:
The byte numbers seems not readable, how about using `util/memory.FormatBytes` to format them?